routes.js

```diff
@@ -0,0 +1,102 @@
+var Todo = require('./models/todo')(Todo);
+
+module.exports = (server) => {
+
+    function errorMessage(testName, error) {
+        console.log("There is an error with " + testName + " : " + error);
+    }
+    function objectNotFound(res, id){
+        res.send('object with id ' + id + ' does not exist.');
+    }
+    //Test 1: Create a Todo
+    server.post('/todos', async (req, res, next) => {
+        try {
+            const listObject = await Todo.create({
+                title: req.body.title,
+                completed: false
+            });
+            res.status(201);
+            var responseObject = {
```

Comment:
Don't define your object and then pass it into `res.send`, just pass it in inline